fudy/NoteScreen/StickNoteModel.cpp

```diff
@@ -157,69 +157,50 @@ bool StickNoteModel::removeCompletedItems() {
 	qCInfo(F_LOG::sticknoteModel) << "Table" << db_utils::cTableName
 							  << "removed all items successfully";
 
+	queryData();
 	return true;
 }
 
-bool StickNoteModel::updateProperty(QString tableName, QString property, int id, QVariant value) {
+bool StickNoteModel::updateProperty(const QString& tableName, const QString& property, int id, const QVariant& value) {
```

Comment:
Please add Todo to refactor them. I think this function is called to much times

Reply:
Of course, thank you.